Directory.Packages.props

```diff
@@ -0,0 +1,30 @@
+<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+    <CentralPackageTransitivePinningEnabled>false</CentralPackageTransitivePinningEnabled>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageVersion Include="ApprovalTests" Version="5.4.7" />
+    <PackageVersion Include="BenchmarkDotNet" Version="0.13.1" />
+    <PackageVersion Include="FluentAssertions" Version="5.10.3" />
+    <PackageVersion Include="Microsoft.CodeAnalysis" Version="4.0.1" />
+    <PackageVersion Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.3" />
+    <PackageVersion Include="Microsoft.CodeAnalysis.CSharp" Version="4.0.1" />
+    <PackageVersion Include="Microsoft.CodeAnalysis.CSharp.Scripting" Version="4.0.1" />
+    <PackageVersion Include="Microsoft.CSharp" Version="4.7.0" />
+    <PackageVersion Include="Microsoft.DotNet.PlatformAbstractions" Version="3.1.6" />
+    <PackageVersion Include="Microsoft.Extensions.Configuration.CommandLine" Version="6.0.0" />
+    <PackageVersion Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="6.0.0" />
+    <PackageVersion Include="Microsoft.Extensions.Hosting" Version="6.0.0" />
+    <PackageVersion Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0" />
+    <PackageVersion Include="Newtonsoft.Json" Version="13.0.2" />
+    <PackageVersion Include="system.reactive.core" Version="5.0.0" />
```

Comment:
It would be good to group those dependencies on where they are coming from. See aspire for an example: https://github.com/dotnet/aspire/blob/main/Directory.Packages.props

Reply:
I see the sample but am not really sure on the best grouping in this case, do you have a preference?

Reply:
the Microsoft.CodeAnalysis ones should be under "roslyn", the Microsoft.Extensions ones should be under "runtime". The others are probably "external".

Reply:
Updated!

Reply:
Just FYI, the Microsoft.NETFramework.ReferenceAssemblies entry will be removed with https://github.com/dotnet/command-line-api/pull/2312/files as it isn't used anymore.

Reply:
> Just FYI, the Microsoft.NETFramework.ReferenceAssemblies entry will be removed with

Removed it, will wait for your pr to merge before rebasing